client/rest-high-level/src/test/java/org/elasticsearch/client/IndexLifecycleIT.java

```diff
@@ -63,31 +59,9 @@
 import static org.elasticsearch.test.hamcrest.ElasticsearchAssertions.assertAcked;
 import static org.hamcrest.Matchers.equalTo;
 import static org.hamcrest.Matchers.hasItems;
-import static org.hamcrest.Matchers.is;
 
 public class IndexLifecycleIT extends ESRestHighLevelClientTestCase {
 
-    public void testSetIndexLifecyclePolicy() throws Exception {
```

Comment:
Might be worth replacing this test case with one that makes sure changing the lifecycle policy via the settings API works as expected (i.e. actually runs the new policy)

Reply:
Agreed, that was the intention of the outstanding task in the description

Reply:
My bad, I misread that task.

Reply:
NP. My intention was to have this as a server side test rather than a test in the HLRC because its testing ILM functionality rather than the client or the API. Does that work for you or do you feel strongly this should be tested as part of the client?

Reply:
I’m good with testing it server side